tcms/testplans/tests/tests.py

```diff
@@ -148,6 +148,12 @@ def test_add_cases_sortkey_autoincrement(self):
         case_plan = plan.add_case(TestCaseFactory(), sortkey=15)
         self.assertEqual(15, case_plan.sortkey)
 
+    def test_get_full_url(self):
+        test_plan_url = self.plan_1.get_full_url()
+        response = self.client.get(test_plan_url)
+        self.assertIsNotNone(response, "Empty response")
+        self.assertNotEqual(response.status_code, 404)
```

Comment:
assert we're on the page for that  TP which will implicitly check status code == 200.